cgatcore/table.py

```diff
@@ -479,135 +479,135 @@ def main(argv=None):
         invert_match=False,
     )
 
-    (options, args) = E.start(parser, add_pipe_options=True)
+    args = E.start(parser, add_pipe_options=True)
```

Comment:
unsure if unknowns should be added here?

Reply:
Yes it should be added because you use stdin further down which isn't defined in argparse